### PR TITLE
Adds more filtering steps to (pre)gwas_filtering

### PR DESCRIPTION
**Pull request by Vlad-Dembrovskyi**
Closes issue #1 
Closes issue #15 

Adds two additional plink commands described by Thanos in branch
kousathanas-support-files. Implements custom file inclusion for plink
--keep argument. Updates Dockerfile to include plink(1) for the added
commands.

Additionaly:
changes --keep to --plink_keep_pheno

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -6,14 +6,17 @@ params {
   phenoFile = false
   phenoCol = false
   qFilter = '0.005:minor'
+  thres_m = '1e-5'
+  thres_HWE = '1e-5'
+  keep = "testdata/nofile"
```
Comment (cgpu) [inline nextflow.config:11]: maybe add more explicit name here in case we need more keeps, eg `plink_keep`

Comment (Vlad-Dembrovskyi) [reply nextflow.config]: changed keep to plink_keep_pheno

**Comment by Vlad-Dembrovskyi** [comment]
Found an issue when running on CloudOS. Will fix it soon.
https://github.com/lifebit-ai/gel-gwas/blob/2c5185dfbc99df79f00e852987a8a67704de174d/nextflow.config#L11
image

**Comment by Vlad-Dembrovskyi** [comment]
Fixed the nodata file path. 
The branch is successfully tested on GEL CloudOS.
image

**Review by cgpu — APPROVED** [review]
Looks good to me @Vlad-Dembrovskyi 👍

**Comment by Vlad-Dembrovskyi** [comment]
The branch was merged independently by @cgpu to main branch. This can be closed.

**Pull request closed without merging**